Gemfile

```diff
@@ -43,3 +43,5 @@ end
 gem "simple_form"
 gem "slim-rails"
 gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
+
+gem "pundit", "~> 2.2"
```

Comment:
What do you think about adding `rubocop-gemfile` gem?

Reply:
Good idea, but I don't think `rubocop-gemfile` is the right way to go. It is in beta, and last update was published 6 years ago.

Reply:
In it's repo it says `USE Bundler/OrderedGems with rubocop v0.46.0+`.